File: javascript/fib.js
const n = 30;
const repeatCount = 10;

function fib(n) {
  return n < 2 ? n : fib(n - 2) + fib(n - 1);
}

const start = Date.now();
for (let i = 0; i < repeatCount; ++i) {
  fib(n);
}
const end = Date.now();

console.log(end - start);

File: c++/fib.cpp
#include <iostream>
#include <chrono>
#include <vector>
using namespace std::chrono;

const int n = 30;
const int repeat_count = 10;

int fib(int n) {
  return n < 2 ? n : fib(n - 2) + fib(n - 1);
}

int main() {
    auto start = system_clock::now();
    auto result = std::vector<int>(); // reject full optimize
    for (auto i = 0; i < repeat_count; ++i) {
        result.push_back(fib(n));
    }
    auto end = system_clock::now();
    std::cout << duration_cast<milliseconds>(end - start).count() << std::endl;
    return 0;
}
